README.md

```diff
@@ -287,3 +288,60 @@ if err != nil {
     t.Error(err)
 }
 ```
+
+## The Encoder
+
+The encoder is used to encode or decode incoming or outgoing data between UDA and the format used in the source we read from or the sink we write to. Example CSV-files, parquet-files or fixed-length-files. The encoder uses the `sourceConfig` JSON object to determine how to encode or decode. 
+
+Example of different `sourceConfig` with descriptions below
+
+### FlatFile-config
+
+```json
+"sourceConfig":{
+    "encoding":"flatfile",
+    "fields":[
+        {
+            "name":"id",
+            "length":10,
+            "ignore": false
+        },
+        {
+            "name":"foo",
+            "length":3,
+            "ignore": false
+        },
+        {
+            "name": "bar",
+            "length":4,
+            "ignore":false
+        }
+
+    ],
+    "indexFrom":2
```

Comment:
Is the start position if there is empty space at the beginning of a line? When we discussed this before I thought we landed on just having another field that could have name omitted, and add this as the first column? This would avoid this additional special case.

Reply:
Could just use the ignore-field already in place and name it, no? That's what I did now, I thought it might be cleaner than adding another check.